package-resources.yml

```diff
@@ -0,0 +1,54 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
+metadata:
+  creationTimestamp: null
+  name: kapp-controller.carvel.dev.0.0.0
+spec:
+  refName: kapp-controller.carvel.dev
+  releasedAt: null
+  template:
+    spec:
+      deploy:
+      - kapp: {}
+      fetch:
+      - git: {}
+      template:
+      - ytt:
+          paths:
+          - config
+          - config-release
+      - kbld: {}
+  valuesSchema:
+    openAPIv3: null
+  version: 0.0.0
+
+---
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: PackageMetadata
+metadata:
+  creationTimestamp: null
+  name: kapp-controller.carvel.dev
+spec:
+  displayName: kapp-controller
+  longDescription: kapp-controller.carvel.dev
+  shortDescription: kapp-controller.carvel.dev
```

Comment:
These ought to reflect something similar to secretgen's descriptions: https://github.com/vmware-tanzu/carvel-secretgen-controller/blob/develop/package-resources.yml#L33-L34

Reply:
Done.